template/build.gradle.kts

```diff
@@ -0,0 +1,57 @@
+plugins {
+    kotlin("multiplatform")
+    id("jacoco")
+    id("maven-publish")
+    id("com.android.library")
+    id("org.jlleitschuh.gradle.ktlint")
+}
+
+val ext = (gradle as ExtensionAware).extra
+
+apply(from = "../gradle/publishable_component.gradle")
+
+group = "com.splendo.kaluga"
+version = ext["library_version"]!!
+
+repositories {
+    google()
+    mavenCentral()
+}
+
+dependencies {
+    implementation("androidx.fragment:fragment:1.1.0")
+    implementation("androidx.appcompat:appcompat:1.1.0")
+    testImplementation("org.mockito:mockito-core:2.28.2")
+    testImplementation("org.junit.jupiter:junit-jupiter-api:5.3.1")
+    testRuntimeOnly("org.junit.jupiter:junit-jupiter-engine:5.3.1")
+    androidTestImplementation("androidx.test.uiautomator:uiautomator:2.2.0")
+    androidTestImplementation("androidx.fragment:fragment-ktx:1.1.0")
+}
+
```

Comment:
Some of these are already in `component.gradle`, remove dupes.

Reply:
Or rather, if some of these are so needed, they should be in components.gradle to begin with. But e.g. I don't agree every project needs fragments.

Reply:
Moved it to component.gradle, disabled fragment. But, because fragments is quite common used dependency I still prefer to have it commented in templete.

Reply:
commented in is a good compromise (in the template then of course). Shall we add a commented line `You can uncomment these frequently used dependencies if you need one` or something` to explain it a bit.